frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -104,7 +104,7 @@
   }
 
   $: userId = user.id;
-  $: canManage = isAdmin(user) || user.projects.find((p) => p.code == project?.code)?.role == 'Manager';
+  $: canManage = isAdmin(user) || project?.users.some(u => u.user.id == userId && u.role == ProjectRole.Manager);
```

Comment:
This is on average 50% more efficient for non-managers 🤷. I know it's not a big deal 😉.
```suggestion
  $: canManage = isAdmin(user) || project?.users.find(u => u.user.id == userId)?.role == ProjectRole.Manager;
```